### DISCUSSION TITLE
Home Assistant OS 14.1.rc1

### DISCUSSION THREAD

**Discussion by sairon** [home-assistant/operating-system#3749] [Announcements]
## Home Assistant Operating System

* Bump Buildroot to 2024.02.9 (#3745) @sairon
* Enable TCPMSS target iptables module for all boards (#3739) @sairon

## Raspberry Pi

* RaspberryPi: Update kernel to 6.6.62 - dd2394360860d15146c96635796a75b05bb32b61 (#3715) @sairon
* Compile hailo_pci module from original sources on RPi 5 (#3732) @sairon
* Fix Hailo-8 firmware path expected since driver v4.19.0 (#3721) @sairon
* Bump RPi SDIO WiFi firmware to latest version (#3719) @sairon
* Bump Hailo-8 firmware to 4.19.0 for RPi kernel 6.6.62 (#3716) @sairon

## Home Assistant Yellow

* RaspberryPi: Update kernel to 6.6.62 - dd2394360860d15146c96635796a75b05bb32b61 (#3715) @sairon
* Remove existing rauc.db from a data disk on the first boot (#3737) @sairon
* Add Hailo PCIe driver and firmware for Yellow (#3734) @sairon
* Bump RPi SDIO WiFi firmware to latest version (#3719) @sairon

## Home Assistant Green

* Linux: Update kernel to 6.6.66 (#3744) @sairon

## Open Virtual Appliance

* Linux: Update kernel to 6.6.66 (#3744) @sairon
* Add Hailo PCIe driver and firmware for x86-64, aarch64 and OVA (#3736) @sairon

## Generic x86-64

* Linux: Update kernel to 6.6.66 (#3744) @sairon
* Add Hailo PCIe driver and firmware for x86-64, aarch64 and OVA (#3736) @sairon

## Hardkernel ODROID

* Linux: Update kernel to 6.6.66 (#3744) @sairon

## ASUS Tinker

* Linux: Update kernel to 6.6.66 (#3744) @sairon

## Khadas VIM Series

* Linux: Update kernel to 6.6.66 (#3744) @sairon

## Generic aarch64

* Linux: Update kernel to 6.6.66 (#3744) @sairon
* Add Hailo PCIe driver and firmware for x86-64, aarch64 and OVA (#3736) @sairon

## Build System

* Add HA Yellow image to RPi Imager index update action (#3731) @sairon
* Bump docker/setup-buildx-action from 3.7.1 to 3.8.0 (#3746) @dependabot

<hr /><em>This discussion was created from the release <a href='https://github.com/home-assistant/operating-system/releases/tag/14.1.rc1'>Home Assistant OS 14.1.rc1</a>.</em>

**Comment by Anto79-ops** [comment]
There are no issues on my NUC 12 (x86-64, bare-metal) with USB SkyConnect and USB Aeotec Z-Stick 7.

The system is running normally.

**Comment by Ladenburg1** [comment]
same issue here on a rpi5 with nvme as with the 14.0. System doesn't reboot and comes after a hard power off/on with the 13.2
